src/utils/chain.ts

```diff
@@ -15,18 +15,43 @@
 import cli from 'cli-ux';
 import { getClient } from '../client';
 
+let blockID: string;
+
 interface ObserveParams {
 	readonly label: string;
 	readonly height: number;
 	readonly liskCorePath: string;
 	readonly delay: number;
+	readonly isFinal: boolean;
 }
 
-export const getChainHeight = async (liskCorePath: string): Promise<number> => {
+export const getChainHeight = async (liskCorePath: string, isFinal: boolean): Promise<number> => {
 	const client = await getClient(liskCorePath);
 	const result = await client.node.getNodeInfo();
+	const height = isFinal ? result.finalizedHeight : result.height;
+	return height;
+};
 
-	return result.height;
+export const setBlockIDAtSnapshotHeight = async (
+	liskCorePath: string,
+	height: number,
+): Promise<string> => {
+	const client = await getClient(liskCorePath);
+	const result = await client.block.getByHeight(height);
+	blockID = result.header.id.toString('hex');
+	return blockID;
+};
+
+export const getBlockIDAtSnapshotHeight = (): string => blockID;
```

Comment:
A setter should not return any value. 
```suggestion
export const setBlockIDAtSnapshotHeight = async (
	liskCorePath: string,
	height: number,
): void => {
	const client = await getClient(liskCorePath);
	const result = await client.block.getByHeight(height);
	blockIDAtSnapshotHeight = result.header.id.toString('hex');
};

export const getBlockIDAtSnapshotHeight = (): string => blockIDAtSnapshotHeight;
```